static/example.html

```diff
@@ -0,0 +1,3323 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8"/>
+<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>chatbot</title>
+<link rel="stylesheet" type="text/css" id="applicationStylesheet" href="/stylesheets/main.css"/>
+<script id="applicationScript">
+///////////////////////////////////////
+// INITIALIZATION
+///////////////////////////////////////
+
+/**
+ * Functionality for scaling, showing by media query, and navigation between multiple pages on a single page. 
+ * Code subject to change.
+ **/
+
+if (window.console==null) { window["console"] = { log : function() {} } }; // some browsers do not set console
+
+var Application = function() {
```

Comment:
Can we separate javascript in separate folder.

Reply:
static/js/ folder would be fine

Reply:
okay!